context/app/static/js/shared-styles/surfaces/Step/Step.tsx

```diff
@@ -34,7 +35,7 @@ function StepDescription({ blocks }: { blocks: (string | ReactElement)[] }) {
     </Stack>
   );
 }
-function Step({ index, title, isRequired = false, children }: PropsWithChildren<StepProps>) {
+function Step({ index, title, isRequired = false, hideRequiredText, children }: PropsWithChildren<StepProps>) {
```

Comment:
Do we need to set a default for `hideRequiredText` to `false` for the other, existing uses of `Step` or does this reflect the designs?

Reply:
If there's no default value set, it's falsy by default due to being `undefined`, so it should fall back to the existing behavior.